homework/shelkovich_ilya/primitiveAndWrapperSpeed/src/test/java/com/tutrit/java/quickstart/service/SumOfPrimitiveAndWrapperTest.java

```diff
@@ -0,0 +1,37 @@
+package com.tutrit.java.quickstart.service;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SumOfPrimitiveAndWrapperTest {
+
+    Sum sum;
+    int a;
+    int b;
+    Integer c;
+    Integer d;
+
+    @Before
+    public void setUp() {
+        this.sum = new Sum();
+        this.a = 10;
+        this.b = 10;
+        this.c = 10;
+        this.d = 10;
```

Comment:
It seems these vars are not used. Could be deleted?

Reply:
deleted redundant code